client/src/components/Modals/SignInModal.tsx

```diff
@@ -0,0 +1,211 @@
+    }, [closeModal]);
+
+    React.useEffect(() => {
+      // This function is called every time the URL changes
+      const unlisten = history.listen(() => {
+        handleCloseModal();
+      });
+
+      // Cleanup function to be run when the component unmounts
+      return () => {
+        unlisten();
+      };
+    }, [history, handleCloseModal]);
+
+    const handleChange = (evt: React.ChangeEvent<HTMLInputElement>): void => {
+      const { name, value }: { name: string; value: string } = evt.target;
+      setFormData((fData) => ({
+        ...fData,
+        [name]: value,
+      }));
+    };
+
+    const handleSubmit = async (evt: React.FormEvent): Promise<void> => {
+      evt.preventDefault();
+      setIsLoading(true);
+      const res = await fetch(`${APP_API_BASE_URL}/auth/login`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(formData),
+        credentials: 'include',
+      });
+      const response = await res.json();
+      setIsLoading(false);
+
+      if (!res.ok) {
+        if (response.error === 'Email not found') {
+          setError({ type: 'email', message: response.error });
+        } else if (response.error === 'Invalid password') {
+          setError({ type: 'password', message: response.error });
+        } else {
+          setError({ type: '', message: 'an unknown error occurred' });
+        }
+      } else {
+        setUser(response.user, false, true);
+        setError(null);
+        history.push('/');
+      }
+    };
+
+    return (
+      <>
+        <Dialog
+          ref={ref}
+          disableEscapeKeyDown={true}
+          open={true}
+          PaperProps={{ className: classes.outerShell }}
+        >
+          <IconButton
+            edge="end"
+            color="inherit"
+            onClick={handleCloseModal}
+            aria-label="close"
+            className={classes.closeButton}
+          >
+            <CloseIcon style={{ fontSize: '1.1em' }} />
+          </IconButton>
+          <DialogContent className={classes.content}>
+            <Paper elevation={0} className={classes.paper}>
+              <Grid container justifyContent="center" direction="column" spacing={2}>
+                <Grid item xs={12}>
+                  <Typography
+                    className={classes.header}
+                    variant="h3"
+                    component={'span'}
+                    align="center"
+                  >
+                    Welcome Back.
+                  </Typography>
+                </Grid>
+                <Grid item xs={12} container justifyContent="space-between" wrap="nowrap">
+                  <Grid item className={classes.buttonContainer}>
+                    <GoogleAuthBtn>Sign In</GoogleAuthBtn>
+                  </Grid>
+                  <Grid item className={classes.buttonContainer}>
+                    <FacebookAuthBtn>Sign In</FacebookAuthBtn>
```

Comment:
We can completely remove the Google and Facebook signin buttons.

Reply:
Removed but left the components just in case